sdcm/sct_events/database.py

```diff
@@ -115,7 +115,8 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
                                    regex="compaction_stopped_exception")
 DatabaseLogEvent.add_subevent_type("BAD_ALLOC", severity=Severity.ERROR,
                                    regex="std::bad_alloc")
-DatabaseLogEvent.add_subevent_type("SCHEMA_FAILURE", severity=Severity.ERROR,
+# Due to scylla issue https://github.com/scylladb/scylladb/issues/19093, we need to suppress the below error
+DatabaseLogEvent.add_subevent_type("SCHEMA_FAILURE", severity=Severity.WARNING,
```

Comment:
Should the `SkipPerIssues` be used here?

Reply:
agree.
```
severity=Severity.ERROR
if SkipPerIssues("scylladb/scylladb#19093", params):
    severity=Severity.WARNING
    
DatabaseLogEvent.add_subevent_type("SCHEMA_FAILURE", severity=severity ........
```

but probably for this, we need to bring the config 
```
from sdcm.sct_config import init_and_verify_sct_config
params=init_and_verify_sct_config()
```